### PR TITLE
feat: change master password

### PR DESCRIPTION
**Pull request by iuricmp**
- Interface for changing the master password.
- Removes `@buf/gnolang_gnonative.bufbuild_es`dependency
- Upgrades gnonative npm package

Signed-off-by: Iuri Pereira <689440+iuricmp@users.noreply.github.com>

### CODE REVIEW

File: mobile/redux/features/signinSlice.ts [hunk 1]
```diff
@@ -48,6 +54,51 @@ export const signIn = createAsyncThunk<boolean, SignInParam, ThunkExtra>("signin
...
+
+    if (response.length === 0) {
+      throw new Error("No accounts found.");
+    }
+
+    for (const account of response) {
+      console.log("change password for account", account);
+      await gnonative.selectAccount(account.name);
+      await gnonative.setPassword(masterPassword);
+      await gnonative.updatePassword(newPassword);
```
Comment (jefft0) [inline mobile/redux/features/signinSlice.ts:82]: Suppose there are 2 keys in the keybase. If the loop updates key 1, but `updatePassword` fails on key 2, then the loop will quit and leave the keybase in an inconsistent state. (Different keys with different passwords.) Do you think this should be some logic to handle this case?

File: mobile/views/change-master-password/index.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+              error={error}
+              secureTextEntry={true}
+              onChangeText={setPassword}
+            />
+            <TextInput
+              placeholder={`Reenter password`}
+              error={error}
+              secureTextEntry={true}
+              onChangeText={setReenterPassword}
+            />
```
Comment (jefft0) [inline mobile/views/change-master-password/index.tsx:89]: image

In most apps, to change the password you have to enter the current password again (to make sure that someone else doesn't pick up your phone and change the password.) Do you think we need confirm the current password here?

Comment (iuricmp) [reply mobile/views/change-master-password/index.tsx:89]: Great idea.

File: mobile/redux/features/signinSlice.ts (hunk 1 above)
Comment (iuricmp) [reply mobile/redux/features/signinSlice.ts:82]: Absolutely.

**Review by jefft0 — APPROVED** [review]
Tested on Android and iOS simulator. In Gno Native Kit, we will implement UpdatePassword for multiple addresses and use this in a later PR.

**Pull request merged**